opensrp-reveal/build.gradle

```diff
@@ -162,13 +162,13 @@ dependencies {
 
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.9'
 
-    implementation('io.ona.kujaku:library:0.8.3') {
+    implementation('io.ona.kujaku:library:0.8.8-ALPHA1') {
```

Comment:
why is alpha version being used

Reply:
This has been updated to use `0.8.8` version